manifest.yml

```diff
@@ -0,0 +1,26 @@
+---
+applications:
+- name: wavefront-firehose-nozzle
+  instances: 1
```

Comment:
`2` is a better default. Never deploy with just 1 instance to avoid downtime (message loss).